Civi/Payment/PropertyBag.php

```diff
@@ -65,6 +65,7 @@ class PropertyBag implements \ArrayAccess {
     'frequency_interval'          => 'recurFrequencyInterval',
     'recurFrequencyUnit'          => TRUE,
     'frequency_unit'              => 'recurFrequencyUnit',
+    'subscriptionId'              => TRUE,
```

Comment:
@eileenmcnaughton @artfulrobot At the very least `subscriptionId` needs to be mapped to a more standard name (eg. `subscriptionID`.  See https://lab.civicrm.org/dev/financial/issues/82#note_27029 and https://lab.civicrm.org/dev/financial/issues/57#note_19168 where this field has been discussed in detail

Reply:
I think we've basically agreed that `civicrm_contribution_recur.processor_id` == `subscriptionID`.

We should update the descriptions in the database as a separate PR:
* processor_id: Possibly needed to store a unique identifier for this recurring payment order - if this is available from the processor??
* trxn_id: unique transaction id. may be processor id, bank id + trans id, or account number + check number... depending on payment_method